BTT/BeyondTheTutor/BeyondTheTutor/Models/Student.cs

```diff
@@ -19,10 +18,10 @@ public partial class Student
         [StringLength(50)]
         public string LastName { get; set; }
 
-        public DateTime GraduatingYear { get; set; }
+        public short GraduatingYear { get; set; }
```

Comment:
Please don't change this... make it into int16 instead of "short" theyre the same thing but i have int16 in like 6 locations :(